### DISCUSSION TITLE
pass req object

### DISCUSSION THREAD

**Discussion by andrewvlgafc** [peers/peerjs#1230] [Ideas & Feature requests]
hello.
Can you pass express req object to use it in peerServer.on('connection'
Need to assess to req.headers to get  x-forwarded-for

**Comment by aarowman** [comment]
Yes! Please add this. This should be easy to add.

It's quite inconvenient if we need to evaluate the request (especially headers or url params for auth) beyond the default peer server interactions. We don't have a way to authenticate the request (server side, not just token field) since the "connection" event doesn't have access to the request object.
